tests/memex/storage_test.py

```diff
@@ -407,10 +434,10 @@ def fetch_annotation(patch):
 
 
 @pytest.fixture
-def models(patch):
-    models = patch('memex.storage.models', autospec=False)
-    models.Annotation.return_value.is_reply = False
-    return models
+def patched_annotation(patch):
+    klass = patch('memex.storage.models.Annotation')
+    klass.return_value.is_reply = False
+    return klass
```

Comment:
Normally, this would work:

```python
import mock
m = mock.MagicMock()  # or mock.Mock()
m.update_document_metadata  # <Mock name='mock.update_document_metadata' id='123'>
m.update_document_metadata.assert_called_once_with('foo')  # AssertionError: Expected 'update_document_metadata' to be called once. Called 0 times.
```

But the `models` fixture in this test file does not work like that, as you say `update_document_metadata.assert_called_once_with('anything')` doesn't raise. But I don't understand why it's apparently not behaving like I expect a mock object to.